realsense2_camera/test/utils/pytest_rs_utils.py

```diff
@@ -770,6 +770,10 @@ def run_test(self, themes):
     def process_data(self, themes):
         for theme in themes:
             data = self.node.pop_first_chunk(theme['topic'])
+            if 'width' in theme:
+                assert theme['width'] == data['shape'][0][1]  # (get from numpy image the width)
```

Comment:
The assert can have a valid message, it will help in debugging when running it in CI or prerelease